packages/salesforcedx-vscode-apex/src/index.ts

```diff
@@ -31,5 +35,13 @@ async function getLineBreakpointInfo(): Promise<{}> {
   return Promise.resolve(response);
 }
 
+async function getExceptionBreakpointInfo(): Promise<{}> {
```

Comment:
Exposed API in salesforcedx-vscode-apex to retrieve a list of System and user-defined exception types.